### DISCUSSION TITLE
A predefined button for BookWyrm

### DISCUSSION THREAD

**Discussion by escuco** [JulianPrieber/littlelink-custom#322] [Feature requests]
Hello! 

An own button for BookWyrm would be nice.

BookWyrm: [joinbookwyrm.com](https://joinbookwyrm.com/)

Logo: [https://joinfediverse.wiki/File:BookWyrm_logo.svg#/media/File:BookWyrm_logo.svg](https://joinfediverse.wiki/File:BookWyrm_logo.svg#/media/File:BookWyrm_logo.svg)

**Comment by MagicLike** [comment]
Hey @escuco and sorry for the late reply...
I opened a PR for this just rn over here: https://github.com/JulianPrieber/littlelink-custom/pull/357

**Reply by MagicLike** [reply]
Resolved in https://github.com/JulianPrieber/littlelink-custom/commit/525245a3565331fc3833d89e26ffed61a0571ec1
